<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="MiniMap" name="_minimap">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="7">
     <widget class="FullImage" name="_image">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="AddLabelBtn" name="_addLabelBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Add label</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="ComputeBtn" name="_computeBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Compute</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="AddGridBtn" name="_addGridBtn">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Add grid</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTableView" name="_properties">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="Tree" name="_tree">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="AutoComputeBtn" name="_autoComputeBtn">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Activate auto compute</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="_actionMade_by_TheCrazyT"/>
    <addaction name="_actionUrl"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuComputation_method">
    <property name="title">
     <string>Computation method</string>
    </property>
    <addaction name="_actionDecisionTree"/>
    <addaction name="_actionNeuralNetwork"/>
   </widget>
   <widget class="QMenu" name="menuWeblinks">
    <property name="title">
     <string>Weblinks</string>
    </property>
    <addaction name="_actionSiliconpr0n"/>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menuComputation_method"/>
   <addaction name="menuWeblinks"/>
  </widget>
  <action name="_actionSaveModel">
   <property name="text">
    <string>Save computation model</string>
   </property>
  </action>
  <action name="_actionLoadModel">
   <property name="text">
    <string>Load computation model</string>
   </property>
   <property name="toolTip">
    <string>Load computation model</string>
   </property>
  </action>
  <action name="_actionRemoveGrid">
   <property name="text">
    <string>Remove grid</string>
   </property>
   <property name="toolTip">
    <string>Remove grid</string>
   </property>
  </action>
  <action name="_actionRemoveLabel">
   <property name="text">
    <string>Remove label</string>
   </property>
   <property name="toolTip">
    <string>Remove label</string>
   </property>
  </action>
  <action name="_actionMade_by_TheCrazyT">
   <property name="text">
    <string>Made by TheCrazyT</string>
   </property>
  </action>
  <action name="_actionUrl">
   <property name="text">
    <string>https://github.com/TheCrazyT/SiliconAnalyser</string>
   </property>
  </action>
  <action name="_actionSaveAsCsv">
   <property name="text">
    <string>Save as csv</string>
   </property>
   <property name="toolTip">
    <string>Save as csv</string>
   </property>
  </action>
  <action name="_actionViewAsPixelimage">
   <property name="text">
    <string>View as pixelimage</string>
   </property>
  </action>
  <action name="_actionExportCellsToImages">
   <property name="text">
    <string>Export cells to images</string>
   </property>
  </action>
  <action name="_actionDecisionTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Decision Tree</string>
   </property>
  </action>
  <action name="_actionNeuralNetwork">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Neural network</string>
   </property>
  </action>
  <action name="_actionGridAddXRowsBottom">
   <property name="text">
    <string>Add X rows to bottom</string>
   </property>
  </action>
  <action name="_actionGridAddXRowsTop">
   <property name="text">
    <string>Add X rows to top</string>
   </property>
   <property name="toolTip">
    <string>Add X rows to top</string>
   </property>
  </action>
  <action name="_actionGridRemoveXRowsBottom">
   <property name="text">
    <string>Remove X rows on bottom</string>
   </property>
   <property name="toolTip">
    <string>Remove X rows on bottom</string>
   </property>
  </action>
  <action name="_actionGridRemoveXRowsTop">
   <property name="text">
    <string>Remove X rows on top</string>
   </property>
   <property name="toolTip">
    <string>Remove X rows on top</string>
   </property>
  </action>
  <action name="_actionSiliconpr0n">
   <property name="text">
    <string>https://siliconpr0n.org/</string>
   </property>
  </action>
  <action name="_actionGridAddXColumnsLeft">
   <property name="text">
    <string>Add X columns to left</string>
   </property>
  </action>
  <action name="_actionGridAddXColumnsRight">
   <property name="text">
    <string>Add X columns to right</string>
   </property>
   <property name="toolTip">
    <string>Add X columns to right</string>
   </property>
  </action>
  <action name="_actionGridRemoveXColumnsLeft">
   <property name="text">
    <string>Remove X columns on left</string>
   </property>
   <property name="toolTip">
    <string>Remove X columns on left</string>
   </property>
  </action>
  <action name="_actionGridRemoveXColumnsRight">
   <property name="text">
    <string>Remove X columns on right</string>
   </property>
   <property name="toolTip">
    <string>Remove X columns on right</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoComputeBtn</class>
   <extends>QPushButton</extends>
   <header>silicon_analyser.helper.autocomputebtn</header>
  </customwidget>
  <customwidget>
   <class>ComputeBtn</class>
   <extends>QPushButton</extends>
   <header>silicon_analyser.helper.computebtn</header>
  </customwidget>
  <customwidget>
   <class>AddLabelBtn</class>
   <extends>QPushButton</extends>
   <header>silicon_analyser.helper.addlabelbtn</header>
  </customwidget>
  <customwidget>
   <class>AddGridBtn</class>
   <extends>QPushButton</extends>
   <header>silicon_analyser.helper.addgridbtn</header>
  </customwidget>
  <customwidget>
   <class>MiniMap</class>
   <extends>QLabel</extends>
   <header>silicon_analyser.helper.minimap</header>
  </customwidget>
  <customwidget>
   <class>FullImage</class>
   <extends>QLabel</extends>
   <header>silicon_analyser.helper.fullimage</header>
  </customwidget>
  <customwidget>
   <class>Tree</class>
   <extends>QTreeView</extends>
   <header>silicon_analyser.helper.tree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
